### ISSUE TITLE
Var 'declared' with local scope.

### ISSUE THREAD

**Issue by ryaneberly** [cflint/CFLint#256]
A var that is first used in assignment in the local scope, and then later unscoped flags a MISSING_VAR message.  It should flag an issue with mixed scope declarations instead (new).

Example:
```
local.foo = 123;
foo = 123;
```
